jigna/core/concurrent.py

```diff
@@ -58,6 +55,7 @@ def do_callback(dispatch, callback, *args):
     """Invoke the callback with a suitable dispatch.
     """
     if dispatch == 'ui':
+        from ..utils.gui import invoke_later
```

Comment:
Instead of repeating the import could this module just have a local set_trait_later function with the import inside of it?